package-builder/flox-build.mk

```
@@ -80,6 +85,10 @@ SHELL := $(_bash)
 # Identify target O/S.
 OS := $(shell $(_uname) -s)
 
+# Nix system
+# TODO(nef): we might be passing that around differntly (or call nef stuff with --impure)
+NIX_SYSTEM = $(shell $(_nix) config show system)
```

```suggestion
NIX_SYSTEM := $(shell $(_nix) config show system)
```
Always use immediate evaluator `:=` to run command at the time of assigning the `NIX_SYSTEM` variable rather than each time it's used.